client/components/chip/test/index.tsx

```diff
@@ -34,24 +47,8 @@ describe( 'Chip', () => {
 		expect( chip ).toMatchSnapshot();
 	} );
 
-	test( 'renders a primary chip by default', () => {
-		const { container: chip } = renderChip( undefined, 'Message' );
-		expect( chip ).toMatchSnapshot();
-	} );
-
 	test( 'renders a warning chip', () => {
 		const { container: chip } = renderChip( 'warning', 'Alert message' );
 		expect( chip ).toMatchSnapshot();
 	} );
-
-	test( 'renders default if type is invalid', () => {
-		const { container: chip } = renderChip( 'invalidtype', 'Message' );
-		expect( chip ).toMatchSnapshot();
-	} );
```

Comment:
The same applies to this test that I mentioned in https://github.com/Automattic/woocommerce-payments/pull/6244/files#r1187915074

Reply:
Did you remove it to avoid TS errors? Because we can safely cast it here just for testing purposes. Something like `'invalid' as ChipType`. As long as we continue using JS, I think it's safer to add/keep those extra tests and conditions.

Reply:
Partly yes, but partly because TS is supposed to get rid of the possibility of even using the component wrong.

Your point about JS still stands though. I'll add the test back in and make sure it's passing 👍

Reply:
I've re-added the tests in b26d5b1419c8e195b70b8bae732eb3989135fc20 🙌